### DISCUSSION TITLE
A contribution does not show up in graph

### DISCUSSION THREAD

**Discussion by Hi-Angel** [community/community#65133] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have a number of contributions to Emacs which virtual assistant claims to have been attributed to my profile. However they never showed up in my profile timeline.

For example: https://github.com/emacs-mirror/emacs/commit/396f46d904ab7509476b0d824ec2e4d9a231a2df This commit dated by March 4, but if you go to my profile and click "March 4" square you won't see anything related to Emacs.

Similar situation with my commits to Mesa. For example this one https://github.com/Mesa3D/mesa/commit/05ca7d114eabfb3e2a2b82ba06ccb7b152e0e2a5 does not appear if you click the square on the timeline.

**Comment by ggdio** [comment]
Hello,

Please check if you have properly configured your local git config with your user data, thats the only way github can associate commits with your user.

Find commands below for configuring your local environment:

`git config --global user.name "Your Name"`
`git config --global user.email "your@email.com"`

**Reply by Hi-Angel** [reply]
Yes I have it configured properly, I have many other commits with these settings to Github-hosted repos.

**Reply by ggdio** [reply]
Hey buddy, I think I know what the problem is, I checked your contributions to emacs, but besides you authored the code changes, who committed it to the repo was someone called Eli-Zaretskii. So in that way, github won't count it as contribution on your personal feed.

**Reply by Hi-Angel** [reply]
This is not it either 😊 I have [this commit](https://github.com/KDE/powerdevil/commit/14647dc6ffd1104c1c0ab393518de8a65cb39d70) from a few days ago which I authored but another person commited, and it appears on my commits graph. So commits which are authored by me but commited by someone else are still attributed to me by Github.

**Reply by ggdio** [reply]
Yeah, ur right.

Maybe you should open a bug ticket reporting this issue to github team.

**Reply by Hi-Angel** [reply]
Bug report ticket? Do you mean there's another bugtracker? I was thinking so far this was exactly what I did, created a bugreport to Github. This page even has a label `Bug`.
